deploy/storage/dev/local-storage/pv-setup/10_daemonset.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: pv-setup
+  labels:
+    app: pv-setup
+spec:
+  selector:
+    matchLabels:
+      app: pv-setup
+  template:
+    metadata:
+      labels:
+        app: pv-setup
+    spec:
+      hostPID: true
+      hostIPC: true
+      hostNetwork: true
+      serviceAccountName: pv-setup
+      containers:
+      - name: pv-setup
+        image: registry.access.redhat.com/ubi8/ubi:latest
+        imagePullPolicy: Always
+        command:
+        - "chroot"
+        - "/host"
+        - "/bin/bash"
+        - "-euExo"
+        - "pipefail"
+        - "-O"
+        - "inherit_errexit"
+        - "-c"
+        - |
+          mkdir -p /var/local-volumes/
+          if [[ ! -f "/var/local-volumes/disk0.xfs" ]]; then
+            dd if=/dev/zero of=/var/local-volumes/disk0.xfs bs=1024 count=0 seek=5G
```

Comment:
I know this is a sparse file but 5T seems like a lot :) I'd go for 10G or something

Reply:
of=/var/persistent-volume-image.xfs
(I don't see the need to index the disk if there is just one and all the volumes are created within the internal filesystem.)